setup.py

```diff
@@ -35,7 +35,7 @@
         "Bug Tracker": "https://github.com/Shopify/feast-trino/issues",
     },
     license="MIT License",
-    packages=["feast_trino"],
+    packages=["feast_trino", "feast_trino.connectors"],
```

Comment:
This should not be necessary as it is a subpackage. Could this be handled instead by adding an `__init__.py` file in `feast_trino.connectors`?

Reply:
Thanks! It seems much more appropriate to add __init__.py file in feast_trino.connectors!

Reply:
Are you comfortable making this change?

Reply:
There was an issue where the `__init__.py` file was added, but the sub-package was not created, so the  `setup.py ` file was partially modified.